lms/static/scripts/frontend_apps/components/LMSFilePicker.js

```diff
@@ -184,10 +221,10 @@ export default function LMSFilePicker({
           in Canvas.
         </p>
       )}
-      {(dialogState.state === 'fetching' ||
-        dialogState.state === 'fetched') && (
+
+      {['fetching', 'fetched'].includes(dialogState.state) && (
```

Comment:
array.prototype.includes was added in ECMA Script 2016, so it should be quite same (without polyfill). Exception is IE, but we don support it.

Reply:
Using `.includes(...)`, and other ES 2016 methods in general, should be fine in the LMS app. We don't have a specific list of minimum versions of major browsers, but a simple rule of thumb is "it should work on all major browsers (Chrome, Firefox, Safari, Edge) released in the last two years".